src/mds/Beacon.cc

```diff
@@ -515,6 +515,14 @@ void Beacon::notify_health(MDSRank const *mds)
                             std::cend(m));
     }
   }
+  if (mds->is_replay()){
```

Comment:
I think we can avoid this warning if replay is short too. Perhaps if the MDS is only in replay for less than 30 seconds, do not raise a warning.

What do you think @vshankar?

Reply:
Good point.

Reply:
Done